src/MarkdownTextInput.web.tsx

```diff
@@ -337,14 +338,14 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
         }
         const changedText = e.target.innerText;
 
-        if (compositionRef.current) {
+        const nativeEvent = e.nativeEvent as MarkdownNativeEvent;
+        if (compositionRef.current && (nativeEvent.inputType !== 'insertCompositionText' || nativeEvent.data !== '*')) {
```

Comment:
Wait!!!

Don't we want to early return if inputType is `insertCompositionText `?

Reply:
@charles-liang I can still reproduce the issue. Can you please test it thoroughly at your end as well?